packages/ilib-loctool-android-layout/package.json

```diff
@@ -40,12 +40,12 @@
     ],
     "repository": {
         "type": "git",
-        "url": "https://github.com/iLib-js/ilib-loctool-android-layout.git"
+        "url": "https://github.com/iLib-js/ilib-mono/ilib-loctool-android-layout.git"
     },
     "scripts": {
-        "dist": "npm pack",
-        "test": "jest",
-        "test:watch": "jest --watch",
+        "dist": "pnpm pack",
+        "test": "pnpm jest",
+        "test:watch": "pnpm jest --watch",
```

Comment:
I think `pnpm ` part optional when you're calling a binary (as in, something placed in `node_modules/.bin`)